README.md

```diff
@@ -96,7 +96,7 @@ The server pod is already running the accelerated `sfnt-pingpong` instance.
 
 Run the client:
 ```
-sh-4.4# LD_PRELOAD=/opt/onload/usr/lib64/libonload.so ./sfnt-pingpong udp 198.19.0.1
+sh-4.4# ./sfnt-pingpong udp 198.19.0.1
 ```
 
 Try running with and without `onload` to compare the reported performance.
```

Comment:
To run without onload you either need to remove the resource specifier in the yaml or prefix command with `LD_PRELOAD= `, which would be simpler for a quick test, I think?

Reply:
I meant to remove that line as well.
Prefixing `LD_PRELOAD=` would only stop the client from being accelerated, it would not affect the server.
I think the neatest way to stop acceleration would be to remove the request for the `amd.com/onload` resource.